pkgs/os-specific/linux/tuxedo-rs/default.nix

```diff
@@ -1,29 +1,38 @@
 { lib
 , fetchFromGitHub
 , rustPlatform
+, testers
+, tuxedo-rs
 }:
 let
 
+  pname = "tuxedo-rs";
```

Comment:
Not sure if deduping `pname` is a good idea, since the repository location might one day change without the package name changing..
Deduping `version` is a good idea, I think.

Reply:
I don't thinks that matters too much, does it?
I mean, if the repo location changes, we have to update the `src` anyway. I'd lean towards YAGNI on this.

Reply:
well either way its not too important